packages/tokamak/sdk/tasks/example-ton-task.env

```diff
@@ -0,0 +1,9 @@
+export PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+export TON=0x0922089E43432516Ab2943AD82464A2880256880
+export ADDRESS_MANAGER=0x7E87BbFd6bbFC8986936CEC7C2Ca6C044CabC981
+export L1_CROSS_DOMAIN_MESSENGER=0x697D6ed516123fb7fe1DF7633eb9d950FEa4337A
```

Comment:
I think we can use hre.deployments.get(~~) instead of address of contracts!
https://github.com/tokamak-network/tokamak-titan-canyon/blob/04b824af3d7dc6658ab35207866d4db2d4787276/packages/tokamak/sdk/tasks/deposit-eth.ts

Reply:
I updated!
The code is also supporting env because I think it is convenient for some usecases we need to override value

Reply:
Oh, It's good idea. Thank you very much! 😍